libtelemetry/src/full/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -413,6 +420,18 @@ private static synchronized void setAccessToken(@NonNull Context context, @NonNu
     }
   }
 
+  @VisibleForTesting
+  static synchronized void resetAccessToken(@NonNull Context context, @NonNull String accessToken) {
```

Comment:
does `lite` not need this?

Reply:
Actually no. This is added here to clear the access token between tests because it is a static field. But can be added in future if required. Change it to resetToken() to make it more meaningful.